### DISCUSSION TITLE
Color schemes

### DISCUSSION THREAD

**Discussion by ghost** [chrisgrieser/shimmering-focus#292] [Q&A]
Hi, sorry if this is a stupid question.
I've been trying to make my own colour schemes, but they keep looking terrible. Has anyone shared any color schemes with you yet? And if so, where can I find them?

Thank you for the theme! From a historian

**Comment by chrisgrieser** [comment]
Not sure I understand this question. There are multiple color schemes shared here in this very discussion forum?
